bats/foreman_helper.bash

```diff
@@ -45,6 +45,12 @@ tSkipIfNoPulp2() {
   fi
 }
 
+tSkipIfPulp2() {
```

Comment:
Wouldn't it be more consistent to use `tSkipIfNoPulp3`

Reply:
For the name of this function? No, as Pulp2 and 3 *can* be co-hosted.
In theory this should be `tSkipIfAnyContentTypesAreWithPulp2` but I didn't want to go and actually implement that ;)

Reply:
Well yes, really it should be `tSkipUnlessPulp3ServesRPM` but that's tricky to determine.

Reply:
Yeah. I'll merge now to unblock @odilhao, and give it a better name tomorrow